src/hooks/useTimerStateMachine.ts

```
@@ -0,0 +1,279 @@
+        status: 'idle',
+        startTime: null,
+        pausedAt: null,
+        phase: action.phase || state.phase,
+        remainingMs: state.totalDurationMs,
+        lastTick: action.timestamp || Date.now(),
+      };
+
+    case 'TICK':
+      if (state.status !== 'running') return state;
+      
+      const elapsed = action.timestamp - (state.startTime || action.timestamp);
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)